pkg/controllers/user/logging/utils/graylog.go

```diff
@@ -0,0 +1,68 @@
+package utils
+
+import (
+	"crypto/tls"
+	"net"
+
+	"github.com/pkg/errors"
+	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
+	"github.com/rancher/types/config/dialer"
+)
+
+var (
+	graylogTestData = []byte(`{ "version": "1.1", "host": "rancher", "short_message": "A short message", "level": 5, "_some_info": "foo" }`)
+)
```

Comment:
better to use the same test info as other targets https://github.com/rancher/rancher/blob/master/pkg/controllers/user/logging/utils/testwrap.go#L35

what `_some_info` used for? Can we remove it?